<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1017</width>
    <height>963</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_5" stretch="0,1">
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_4">
        <property name="sizeConstraint">
         <enum>QLayout::SetMaximumSize</enum>
        </property>
        <item>
         <widget class="QPushButton" name="pushButton_4">
          <property name="text">
           <string>Druck(Winkel)</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="pushButton_5">
          <property name="text">
           <string>Testdaten</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="pushButton_6">
          <property name="text">
           <string>Abstand Linie Punkt</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="pushButton_7">
          <property name="toolTip">
           <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Kurve zufällig ändern und ständig wiederholen&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
          </property>
          <property name="text">
           <string>Zufall</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="pushButton_8">
          <property name="toolTip">
           <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Punktanzahl reduzieren&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
          </property>
          <property name="text">
           <string>Optimierung</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QStackedWidget" name="stackedWidget">
          <property name="currentIndex">
           <number>1</number>
          </property>
          <widget class="QWidget" name="page_4"/>
          <widget class="QWidget" name="page">
           <widget class="QPushButton" name="pushButton_10">
            <property name="geometry">
             <rect>
              <x>0</x>
              <y>150</y>
              <width>141</width>
              <height>31</height>
             </rect>
            </property>
            <property name="text">
             <string>Optimierung Ralph</string>
            </property>
           </widget>
           <widget class="QPushButton" name="pushButton_11">
            <property name="geometry">
             <rect>
              <x>0</x>
              <y>70</y>
              <width>141</width>
              <height>31</height>
             </rect>
            </property>
            <property name="text">
             <string>Timer</string>
            </property>
           </widget>
           <widget class="QPushButton" name="pushButton_12">
            <property name="geometry">
             <rect>
              <x>0</x>
              <y>110</y>
              <width>141</width>
              <height>31</height>
             </rect>
            </property>
            <property name="text">
             <string>Optimierung DP</string>
            </property>
           </widget>
           <widget class="QWidget" name="horizontalLayoutWidget_3">
            <property name="geometry">
             <rect>
              <x>0</x>
              <y>20</y>
              <width>161</width>
              <height>41</height>
             </rect>
            </property>
            <layout class="QHBoxLayout" name="horizontalLayout_3" stretch="4,1,1">
             <item>
              <widget class="QLabel" name="label_2">
               <property name="text">
                <string>Schrittweite</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLineEdit" name="schrittweite1">
               <property name="text">
                <string>0.5</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QPushButton" name="pushButton_9">
               <property name="text">
                <string>OK</string>
               </property>
              </widget>
             </item>
            </layout>
           </widget>
          </widget>
          <widget class="QWidget" name="page_2">
           <widget class="QLineEdit" name="lineEdit_3">
            <property name="geometry">
             <rect>
              <x>10</x>
              <y>40</y>
              <width>81</width>
              <height>20</height>
             </rect>
            </property>
           </widget>
          </widget>
          <widget class="QWidget" name="page_3">
           <widget class="QWidget" name="horizontalLayoutWidget_2">
            <property name="geometry">
             <rect>
              <x>0</x>
              <y>60</y>
              <width>281</width>
              <height>31</height>
             </rect>
            </property>
            <layout class="QHBoxLayout" name="horizontalLayout_2" stretch="1,4">
             <property name="spacing">
              <number>4</number>
             </property>
             <item>
              <widget class="QLabel" name="Abschtanstext">
               <property name="text">
                <string>Linienlänge :</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLabel" name="linienlaenge">
               <property name="text">
                <string/>
               </property>
              </widget>
             </item>
            </layout>
           </widget>
           <widget class="QWidget" name="horizontalLayoutWidget">
            <property name="geometry">
             <rect>
              <x>0</x>
              <y>30</y>
              <width>131</width>
              <height>31</height>
             </rect>
            </property>
            <layout class="QHBoxLayout" name="horizontalLayout">
             <item>
              <widget class="QLabel" name="label">
               <property name="text">
                <string>Abstand : </string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLabel" name="abstand">
               <property name="text">
                <string/>
               </property>
              </widget>
             </item>
            </layout>
           </widget>
          </widget>
          <widget class="QWidget" name="page_5"/>
         </widget>
        </item>
        <item>
         <spacer name="verticalSpacer">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>30</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QPushButton" name="diagrammtypUmschalter">
          <property name="text">
           <string>Diagrammtyp umschalten</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="pushButton_3">
          <property name="text">
           <string>Maximieren</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="pushButton_2">
          <property name="text">
           <string>Verkeinern</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="pushButton">
          <property name="text">
           <string>Schließen</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <widget class="MyLineChart" name="graphicsView">
        <property name="baseSize">
         <size>
          <width>500</width>
          <height>500</height>
         </size>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1017</width>
     <height>17</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>MyLineChart</class>
   <extends>QGraphicsView</extends>
   <header location="global">mylinechart.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections>
  <connection>
   <sender>pushButton</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>107</x>
     <y>932</y>
    </hint>
    <hint type="destinationlabel">
     <x>123</x>
     <y>339</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton_3</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>showMaximized()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>62</x>
     <y>892</y>
    </hint>
    <hint type="destinationlabel">
     <x>73</x>
     <y>248</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton_2</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>showNormal()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>107</x>
     <y>912</y>
    </hint>
    <hint type="destinationlabel">
     <x>205</x>
     <y>243</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
